src/components/Home.vue

```diff
@@ -73,19 +69,19 @@ function getNextTask() {
                   <tbody>
                     <tr>
                       <td>Tool Name</td>
-                      <td>{{ currentTask.toolName }}</td>
+                      <td>{{ currentTask.tool.name }}</td>
```

Comment:
I think we should  use null checks to account for the situation whereby maybe due to a network issue the currentTask object is unavailable or the slight off chance that the nested property is unavailable from the BE.

This can be done by either adding a conditional check to the v-table element like this:
```<v-table class="table elevation-2" v-if="currentTask">```
or adding a ternary null check to the cells reading from the currentTask object, like this:
```currentTask?.tool?.name```

Failure to do this could cause the page to break when the network is out.

Reply:
works perfectly after adding the ternary null check. Thank you